src/components/HistoryTable/row/LoadingRow.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from "@emotion/styled";
+import { Skeleton } from "antd";
+import { v4 as uuid } from "uuid";
+import { LoadingCell } from "../Cell";
+
+interface LoadingRowProps {
+  numVisibleCols: number;
+}
+export const LoadingRow: React.FC<LoadingRowProps> = ({ numVisibleCols }) => (
+  <>
+    <LabelCellContainer>
+      <Skeleton active title={false} paragraph={{ rows: 3 }} />
+    </LabelCellContainer>
+    {Array.from(Array(numVisibleCols)).map(() => (
+      <LoadingCell key={`loading_row_${uuid()}`} />
```

Comment:
I added `uuid` to the key here because I was concerned that there would be multiple duplicate keys for these cells
Maybe there is a better way to do this 🤔

Reply:
uuid will change the key on every render and isn't useful as a key. You will be better off using the array index as the key in this case. Since we are rendering the same element every time it would be ok.